### DISCUSSION TITLE
Teslamte stop working "waiting for postgres at database"

### DISCUSSION THREAD

**Discussion by CptCheesyCrust** [teslamate-org/teslamate#3150] [Q&A]
Hey,

Without me doing anything, Teslmate stopped recording things yesterday. I can still access the Webview (Teslamate is running on a RaspberryPi4) and also browse and view all trips and loads, but the first page is the latest status from yesterday and nothing has been recorded since then. 

Teslamate Log:

> teslamate_1  | 2023-03-13 08:54:57.766 [notice] SIGTERM received - shutting down
teslamate_1  | 
teslamate_1  | 2023-03-13 08:54:58.599 [error] Postgrex.Protocol (#PID<0.2388.0>) disconnected: ** (Postgrex.Error) FATAL 57P01 (admin_shutdown) terminating connection due to administrator command
teslamate_1  | 2023-03-13 08:54:58.600 [error] Postgrex.Protocol (#PID<0.2389.0>) disconnected: ** (Postgrex.Error) FATAL 57P01 (admin_shutdown) terminating connection due to administrator command
teslamate_1  | 2023-03-13 08:54:58.600 [error] Postgrex.Protocol (#PID<0.2382.0>) disconnected: ** (Postgrex.Error) FATAL 57P01 (admin_shutdown) terminating connection due to administrator command
teslamate_1  | 2023-03-13 08:54:58.601 [error] Postgrex.Protocol #PID<0.2382.0> could not cancel backend: tcp connect: connection refused - :econnrefused
teslamate_1  | 2023-03-13 08:54:58.602 [error] Postgrex.Protocol #PID<0.2388.0> could not cancel backend: tcp connect: connection refused - :econnrefused
teslamate_1  | 2023-03-13 08:54:58.602 [error] Postgrex.Protocol #PID<0.2389.0> could not cancel backend: tcp connect: connection refused - :econnrefused
teslamate_1  | 2023-03-13 08:54:58.605 [error] Postgrex.Protocol (#PID<0.2389.0>) failed to connect: ** (DBConnection.ConnectionError) tcp connect (database:5432): connection refused - :econnrefused
teslamate_1  | 2023-03-13 08:54:58.605 [error] Postgrex.Protocol (#PID<0.2388.0>) failed to connect: ** (DBConnection.ConnectionError) tcp connect (database:5432): connection refused - :econnrefused
teslamate_1  | 2023-03-13 08:54:58.605 [error] Postgrex.Protocol (#PID<0.2382.0>) failed to connect: ** (DBConnection.ConnectionError) tcp connect (database:5432): connection refused - :econnrefused
teslamate_1  | 2023-03-13 08:54:59.598 [error] Postgrex.Protocol (#PID<0.2392.0>) disconnected: ** (Postgrex.Error) FATAL 57P01 (admin_shutdown) terminating connection due to administrator command
teslamate_1  | 2023-03-13 08:54:59.598 [error] Postgrex.Protocol (#PID<0.2386.0>) disconnected: ** (Postgrex.Error) FATAL 57P01 (admin_shutdown) terminating connection due to administrator command
teslamate_1  | 2023-03-13 08:54:59.599 [error] Postgrex.Protocol (#PID<0.2385.0>) disconnected: ** (Postgrex.Error) FATAL 57P01 (admin_shutdown) terminating connection due to administrator command
teslamate_1  | 2023-03-13 08:54:59.599 [error] Postgrex.Protocol (#PID<0.2390.0>) disconnected: ** (Postgrex.Error) FATAL 57P01 (admin_shutdown) terminating connection due to administrator command
teslamate_1  | 2023-03-13 08:54:59.600 [error] Postgrex.Protocol (#PID<0.2387.0>) disconnected: ** (Postgrex.Error) FATAL 57P01 (admin_shutdown) terminating connection due to administrator command
teslamate_1  | 2023-03-13 08:54:59.600 [error] Postgrex.Protocol (#PID<0.2381.0>) disconnected: ** (Postgrex.Error) FATAL 57P01 (admin_shutdown) terminating connection due to administrator command
teslamate_1  | 2023-03-13 08:54:59.600 [error] Postgrex.Protocol (#PID<0.2391.0>) disconnected: ** (Postgrex.Error) FATAL 57P01 (admin_shutdown) terminating connection due to administrator command
teslamate_1  | 2023-03-13 08:54:59.750 [error] Postgrex.Protocol (#PID<0.2388.0>) failed to connect: ** (DBConnection.ConnectionError) tcp connect (database:5432): non-existing domain - :nxdomain
teslamate_1  | 2023-03-13 08:55:00.139 [error] Postgrex.Protocol (#PID<0.2389.0>) failed to connect: ** (DBConnection.ConnectionError) tcp connect (database:5432): non-existing domain - :nxdomain
teslamate_1  | 2023-03-13 08:55:00.680 [error] Postgrex.Protocol (#PID<0.2382.0>) failed to connect: ** (DBConnection.ConnectionError) tcp connect (database:5432): non-existing domain - :nxdomain
teslamate_1  | 2023-03-13 08:55:01.306 [error] Postgrex.Protocol (#PID<0.2388.0>) failed to connect: ** (DBConnection.ConnectionError) tcp connect (database:5432): non-existing domain - :nxdomain
teslamate_1  | waiting for postgres at database:5432
teslamate_1  | waiting for postgres at database:5432
teslamate_1  | 2023-03-13 08:56:00.512 [info] Migrations already up
teslamate_1  | 2023-03-13 08:56:05.474 [info] System Info: Erlang/OTP 24 (emu)
teslamate_1  | 2023-03-13 08:56:05.474 [info] Version: 1.27.2
teslamate_1  | 2023-03-13 08:56:06.040 [info] POST https://auth.tesla.com/oauth2/v3/token -> 200 (463.934 ms)
teslamate_1  | 2023-03-13 08:56:06.041 [info] Refreshed api tokens
teslamate_1  | 2023-03-13 08:56:06.052 [info] Scheduling token refresh in 6 h
teslamate_1  | 2023-03-13 08:56:06.059 [info] Running TeslaMateWeb.Endpoint with cowboy 2.9.0 at :::4000 (http)
teslamate_1  | 2023-03-13 08:56:06.070 [info] Access TeslaMateWeb.Endpoint at http://localhost
teslamate_1  | 2023-03-13 08:56:06.334 [info] Starting logger for ''
teslamate_1  | 2023-03-13 08:56:06.472 [info] MQTT connection has been established
teslamate_1  | 2023-03-13 08:56:06.681 car_id=3 [info] Start / :asleep
teslamate_1  | 2023-03-13 08:56:08.264 [info] GET /
teslamate_1  | 2023-03-13 08:56:08.287 [info] Sent 200 in 22ms

Maybe someone has an idea? :-(

**Comment by DrMichael** [comment]
Looks like a system issue, no name resolution. But at the end, it looks ok?

**Comment by CptCheesyCrust** [comment]
Strange. I was just getting something out of the trunk of the car. Now Teslamate is showing the current status again?!?

However, he did not record the 200km journey and the charge in the evening. 🤔
